addons/http_routing/controllers/main.py

```diff
@@ -9,7 +9,7 @@
 class Routing(Home):
 
     @http.route('/website/translations/<string:unique>', type='http', auth="public", website=True)
-    def get_website_translations(self, unique, lang, mods=None):
+    def get_website_translations(self, unique, lang=None, mods=None):
```

Comment:
beware of commit message ([REF] web, website)

Reply:
also, I'll trust you on this, no idea why you need to change

Reply:
I am a bit more curious, why that change? :wink:

Reply:
Because the `WebClient` method which we call has this param marked as optional, so in the localization service, when we call the translation route, if the lang is not present, we don't put the param in the URL, whereas in the previous code for the frontend, we would always add it to the URL, and if not present it would be an empty string. Instead of trying to share a translation object globally through some dirty tricks (like setting a promise on the global odoo object), we decided to leverage browser caching, so the URLs have to be the same, and I thought it made more sense to make the parameter optional (functionally, it already is, it does the same thing when you pass an empty string as when you don't pass it at all) rather than always putting the lang param in the URL even if it's empty.